.kokoro/autorelease.sh

```diff
@@ -27,63 +27,26 @@ git config core.fileMode false
 git config user.name "$github_user"
 git config user.email "$github_email"
 
-# Download, generate, build and pack all generated libraries
-# Build support libraries in case the latest support library version isn't yet on nuget.
-
-./BuildSupport.sh
```

Comment:
This means that we now have to always manually release support. I'm fine with that, just calling it out.

Reply:
Yes, that was conscious and we should document it - not as a change, but in terms of "this is the process for a support release".